python/cudf/cudf/core/dataframe.py

```diff
@@ -1531,7 +1531,7 @@ def reset_index(self, drop=False, inplace=False):
         else:
             return result
 
-    def take(self, positions, index=True):
```

Comment:
I am not sure I follow. `Series` handles the index [separately](https://github.com/rapidsai/cudf/blob/branch-0.14/python/cudf/cudf/core/series.py#L454-L456) already and passes the column as an indexless `Table` to `gather`.

Reply:
https://github.com/rapidsai/cudf/blob/branch-0.14/python/cudf/cudf/core/series.py#L481-L484

This will always gather the index as well and doesn't give the opportunity to bypass it as an optimization.

Reply:
@kkraus14 I am still planning on getting this in for this PR, hopefully today.